requirements.txt

```diff
@@ -3,7 +3,7 @@ addict==2.1.0
 aiohttp==3.7.4.post0
 argcomplete==0.8.1
 argparse==1.4.0
-arrow==0.12.1
+arrow==1.1.0
```

Comment:
0.12.1 was never used internally since we had things in our yelpy requirements that required newer versions